Libraries/Hotcakes.Payment/Hotcakes.Payment.csproj

```diff
@@ -68,8 +68,8 @@
     <Reference Include="Microsoft.Bcl.AsyncInterfaces, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
       <HintPath>..\..\packages\Microsoft.Bcl.AsyncInterfaces.1.1.0\lib\net461\Microsoft.Bcl.AsyncInterfaces.dll</HintPath>
     </Reference>
-    <Reference Include="Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Newtonsoft.Json.9.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json">
```

Comment:
@VAlmea Shouldn't this just been an update to the nuget version of the assembly, instead of pointing at a local assembly?  DNN hasn't used Newtonsoft.Json version lower than 10 for a while now.

Reply:
@VAlmea I don't see a response to this question.  Can you please let me know if this is something we need to chat about more?

Reply:
fixed